ups/deploy_testing_distrib.cfg

```diff
@@ -0,0 +1,13 @@
+# -*- python -*-
+
+import lsst.sconsUtils
+
+dependencies = dict(
+)
+
+config = lsst.sconsUtils.Configuration(
+    __file__,	
+    libs=[],
+    hasDoxygenInclude=False,
+    hasSwigFiles=False,
+    )
```

Comment:
(Moving my comment). `cfg` files are not needed for metapackages as they exist to teach `sconsUtils` how to link against C++ libraries).